### ISSUE TITLE
Inconsistent usage of the age field in the PsCookie class.

### ISSUE THREAD

**Issue by danff** [yegor256/takes#561]
The javadoc in the PsCookie class for field **age** is commenting as "Max login age, in seconds.".
One ctor for the class has a parameter sec commented as "Max age in seconds" which is assigned to the **age** field.
But the class is using the field as a number of **days** instead of seconds as the comments say when building the cookie "Expires" part.
```
            String.format(
                Locale.ENGLISH,
                "Expires=%1$ta, %1$td %1$tb %1$tY %1$tT GMT",
                new Date(
                    System.currentTimeMillis()
                        + TimeUnit.DAYS.toMillis(this.age)
                )
            )
```
Either the comments should be corrected or the expiration should be calculated based on seconds count instead of days.

**Comment by davvd** [comment]
@yegor256 please take a look at it and dispatch (see [par.21](http://at.teamed.io/policy.html#21))

**Comment by yegor256** [comment]
@danff fixed in a749f6a thanks!
